go.mod

```diff
@@ -89,6 +89,7 @@ require (
 	goji.io v2.0.2+incompatible
 	golang.org/x/image v0.12.0
 	golang.org/x/sys v0.13.0
+	golang.org/x/term v0.13.0
```

Comment:
Ah this does add a direct dependency to the RDK; any special protocol there @Otterverse? In my experience, a transitive dependency (which `golang/x/term` was previously) is not wildly distinct from a direct one.

Reply:
Yeah, the code is in our deps either way, and it's a golang.org one so we can expect it will be maintained pretty reliably, so no issues with it.

Reply:
Sounds good; thanks.